CeedlingCreateProject.py

```diff
@@ -13,6 +11,11 @@ def run(self, options=[]):
 
         window = self.window
         view = self.window.active_view()
+        active = sublime.active_window()
+
+        if active.views() or active.project_data():
+            sublime.error_message("New Project requires a new, empty window.")

```

Comment:
Alternatively you could also do it like this:

https://github.com/SublimeText/PackageDev/blob/742c8154f15d1f9fdb2bc868ca4a4b086ac304c6/plugins/create_package.py#L47

Reply:
Much neater way of doing it. Have updated.